_data/repos.json

```diff
@@ -497,11 +497,11 @@
       "homepage": ""
     },
     {
-      "name": "Scap Security Guide (SSG)",
+      "name": "ComplianceAsCode Project",
```

Comment:
The https://github.com/ComplianceAsCode/content repo's metadata identifies its website as https://www.open-scap.org/security-policies/scap-security-guide/ 
At that linked website, it self-identifies as "SCAP Security Guide".

Reply:
Can't update everything at once, unfortunately.

Reply:
I'm happy to adhere to whatever the project itself names itself. But I think it would make sense to keep it consistent in multiple places. For now, the website that repo points to still identifies itself as "SCAP Security Guide". Until it identifies itself as "ComplianceAsCode Project" in some way on its own website, I think changing it here would be premature, as the inconsistency could be confusing.